gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineIdeaIntegrationTest.groovy

```diff
@@ -49,7 +49,23 @@ class BaselineIdeaIntegrationTest extends AbstractPluginTest {
         buildFile << standardBuildFile
 
         then:
-        with('idea').build()
```

Comment:
this assertion would always pass. the result contained an outcome of either success/fail, but we never checked it. 🤦